testing/functional-tests.yaml

```diff
@@ -3,7 +3,8 @@ resources:
     desired: pass
     script:
     manifests:
-      - test-deploy01.yaml
+      - file: test-deploy01.yaml
+        name: "Deployment: Pass: 1"
```

Comment:
Would like to leave this as it was or work with you to flesh out the proper descriptions for each test.

Reply:
Backed it out for now so we're not blocked if we don't find time to craft a new manifest file. Added a note to the PR about how to modify the manifests list to enable the new feature.